hw/ip/dma/data/dma.hjson

```diff
@@ -33,13 +33,32 @@
         { bits: "31:0", name: "PTR_ADDR", desc: "Address data pointer (word aligned) - used only in Address mode" }
       ]
     },
-    { name:     "SIZE",
-      desc:     "Number of bytes to copy - Once a value is written, the copy starts",
+    { name:     "SIZE_TR_D1",
+      desc:     "Number of bytes to copy from, defined with respect to the first dimension - Once a value is written, the copy starts",
       swaccess: "rw",
       hwaccess: "hro",
       hwqe:     "true", // enable `qe` latched signal of software write pulse
       fields: [
-        { bits: "31:0", name: "SIZE", desc: "DMA counter and start" }
+        { bits: "31:0", name: "SIZE", desc: "DMA counter D1 and start" }
+      ]
+    },
+    { name:     "SIZE_TR_D2",
+      desc:     "Number of bytes to copy from, defined with respect to the second dimension",
+      swaccess: "rw",
+      hwaccess: "hro",
+      hwqe:     "true", // enable `qe` latched signal of software write pulse
+      fields: [
+        { bits: "31:0", name: "SIZE", desc: "DMA counter D2" }
```

Comment:
let's make it to copy max 64kB, so 16bit size in byte should be enough

Reply:
Done